.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
It should run natively on vim.

Em quarta-feira, 14 de outubro de 2015, Carlos Oliveira <
carlospecter@gmail.com> escreveu:

> Are you using vim or vi?
>
> Em quarta-feira, 14 de outubro de 2015, Rodrigo Siqueira de Melo <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> escreveu:
>
>> In .vimrc
>> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11#discussion_r41994118>
>> :
>>
>> > @@ -84,8 +84,10 @@ set cursorline
>> >
>> >  ""------------ Mappings --------------"
>> >  "" Create tree navigation
>> > -map <C-n> :vsp . <bar> vertical resize 30 <cr>
>> > +map <C-n> :Lexplore <CR>
>>
>> Not an editor command: Lexplore. Something like that. i changed to
>> "Explore" and it works, however worked as "vsp"
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11/files#r41994118>
>> .
>>
>
>
> --
> Carlos Oliveira
> LAPPIS - UnB/FGA(https://lappis.unb.br/)
> GitHub: @carloshfoliveira <https://github.com/carloshfoliveira>
> LinkedIn: @carloshenriquef <https://br.linkedin.com/in/carloshenriquef>
>
>

-- 
Carlos Oliveira
LAPPIS - UnB/FGA(https://lappis.unb.br/)
GitHub: @carloshfoliveira <https://github.com/carloshfoliveira>
LinkedIn: @carloshenriquef <https://br.linkedin.com/in/carloshenriquef>